packages/ui/react-ui/package.json

```diff
@@ -32,7 +32,7 @@
         "test": "jest"
     },
     "dependencies": {
-        "@crossmint/client-sdk-base": "^0.2.1-alpha.0",
+        "@crossmint/client-sdk-base": "file:../../core/base",
```

Comment:
maybe we want to take this back to what it was?

Reply:
Definetly, my question here: how does this work  on this specific case? 
UI needs CORE changes that are also submitted on this PR